benchmarks/dashboard-stack/lib/dashboard-stack.ts

```diff
@@ -201,5 +229,12 @@ export class DashboardStack extends cdk.Stack {
             projectName: "S3BenchmarksDeploy"
         });
 
+        // Create the Key Pair
+        const key = new KeyPair(this, 'EC2CanaryKeyPair', {
```

Comment:
Not necessary to change now (or maybe ever), but I think it would be okay to have this passed in as an argument to the stack as a pre-existing resource.

Reply:
hmmm, I think It's not bad to have it. Any reason we want to remove this?